src/Sulu/Component/Webspace/Analyzer/Attributes/PortalInformationRequestProcessor.php

```diff
@@ -66,12 +66,7 @@ public function process(Request $request, RequestAttributes $requestAttributes)
         $attributes['resourceLocator'] = $resourceLocator;
         $attributes['format'] = $format;
 
-        $host = $request->getHost();
-        if ($portalInformation->getType() !== RequestAnalyzerInterface::MATCH_TYPE_WILDCARD) {
-            $urlInfo = parse_url($request->getScheme() . '://' . $portalInformation->getUrl());
```

Comment:
@wachterjohannes Any idea why you have written this like it was? Shouldn't the URL from the `portalInformation` match the current host anyway?